_posts/guides/2019-08-27-guide_the-pbr-shader.md

```diff
@@ -59,23 +63,73 @@ PBR
     </p>
 </div>
 
+## VMT Example - Brush
+Here is another example using parallax, intended for use on a brush:
+```
+PBR
+{
+	$basetexture "example\pbr_asset_d"
+	$mraotexture "example\pbr_asset_mrao"
+	$bumpmap "example\pbr_asset_n"
+	
+	$envmap "env_cubemap"
+	$parallax "1"
+	$parallaxdepth "0.03"
+	$parallaxcenter "0.0"
+}
+```
+
+<div class="note info">
+    <p>
+        The Parallax option requires that a Heightmap is embedded into the Alpha Channel of the Normal Map.  
+        You can check out one way of doing so <a href="#texture-creation---designer">below</a>.
+    </p>
+</div>
+
+`$parallax` simply enables the Parallax effect.
+`$parallaxdepth` specifies how “deep” the parallax effect will go.
+`$parallaxcenter` adjusts the centre-point of the parallax effect, nudging it closer or further away from the surface.
+
+<div class="note warning">
+    <p>
+        Parallax Occlusion has its limitations and things to be wary of.<br><br>
```

Comment:
```suggestion
        Parallax pcclusion has its limitations and things to be wary of.<br><br>
```

Reply:
I think in this case the capitalisation of Occlusion is fine because it's intended to refer to the technique itself, at least in this implementation that the shader uses.

Reply:
Sounds good, spelt it wrong anyways :p